### DISCUSSION TITLE
Issue with Running  Monorepo with NX Nextjs Build on Jenkins Docker Environment

### DISCUSSION THREAD

**Discussion by ayazsaif** [nrwl/nx#22550] [General]
Hello everyone,

I'm encountering an issue while trying to build an image on Jenkins using Docker for an NX Next.js Monorepo. The build process is working fine when running on my local Docker environment, but it's failing on Jenkins with the following error message:
```
NX   The current directory isn't part of an Nx workspace.

To create a workspace run:
npx create-nx-workspace@latest <workspace name>

To add Nx to an existing workspace with a workspace-specific nx.json, run:
npx nx@latest init
```

I'm using the following plugins in my NX workspace:

```
@nx/cypress (executors, generators)
@nx/eslint (executors, generators)
@nx/eslint-plugin
@nx/jest (executors, generators)
@nx/js (executors, generators)
@nx/next (executors, generators)
@nx/node (generators)
@nx/workspace (executors, generators)
nx (executors, generators)
```

Here's the Dockerfile configuration I'm using:

```
FROM node:18.17.0-alpine

RUN yarn global add nx 

ADD package*.json yarn.* nx.json ./

RUN yarn install

COPY . .

RUN nx run project:build

EXPOSE 3003
```

I'm using NX version : "^18.1.3".

Could someone please assist me in resolving this issue? Thank you!

**Comment by p4cino** [comment]
some things you need to add start with 
https://github.com/gperdomor/nx-tools/tree/main/packages/nx-container
+
https://nextjs.org/docs/app/building-your-application/deploying#docker-image

you need to set standalone as export and then set the paths in docker image
